### PR TITLE
Feature. Add timer. Closes: #9

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -48,7 +55,10 @@ class Game {
         this.imageContainers[this.prevClicked].removeEventListener('click', this.imageOnClickBinded);
         this.prevClicked = undefined;
         this.nowOpened = 0;
-
+        this.numCards-=2;
+        if(!this.numCards) {
+            clearInterval(this.interval);
```
Comment (probil) [inline main.js]: Extract this as `stopTimer` method on `Timer` class.
So this line will become:
```js
this.timer.stopTimer();
```

File: main.js [hunk 2]
```diff
@@ -27,6 +29,11 @@ class Game {
         }
     }
     imageOnClick(event) {
+        if(!this.interval) {
+            let timer=new Timer();
+            this.interval = timer.startTimer();
```
Comment (probil) [inline main.js]: `Game` class should not know how `Timer` is implemented. 
Implementation may vary. `setInterval` is only one approach
I think it's better to keep instance of timer class inside the `Game` class instead.
And do all the manipulations by calling Timer's methods.

File: main.js [hunk 3]
```diff
@@ -58,6 +68,23 @@ class Game {
...
+        return setInterval(function(){
+            numSec++;
+            let minutes=Math.floor(numSec/60);
+            if(minutes<10) minutes='0'+ minutes;
+            let seconds=numSec % 60;
+            if(seconds<10) seconds='0'+ seconds;
+            let timerValue=`${minutes}:${seconds}`;
+            pageTimer.firstChild.data=timerValue;
+        },1000);
+    }
```
Comment (probil) [inline main.js]: Let's add
- `stopTimer` method which will stop the setInterval by `intervalId`
- `isStarted` [**getter** ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)that will return status of the timer
And use them in `Game` class

File: main.js [hunk 4]
```diff
@@ -27,6 +28,11 @@ class Game {
         }
     }
     imageOnClick(event) {
+        if(!this.timer || !this.timer.isStarted) {
+            this.timer=new Timer();
```
Comment (probil) [inline main.js]: Let's move instance creation to the constructor and then simplify the code
```js
if(!this.timer.isStarted) {
  this.timer.startTimer();
}
```

Comment (liubich) [reply main.js]: @probil, ok.

**Pull request merged**